### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by therealklanni**
Add webpack config and dependencies

For #25

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,16 @@
   "devDependencies": {
     "applitude": "~0.x.x",
     "grunt": "~0.3.x"
```
Comment (ericelliott) [inline package.json]: missing comma?

File: package.json [hunk 2]
```diff
@@ -29,8 +29,16 @@
...
+    "babel-loader": "^5.3.0",
+    "node-libs-browser": "^0.5.2",
+    "rimraf": "^2.4.1",
+    "webpack": "^1.10.1"
   },
   "scripts": {
     "test": "grunt test"
```
Comment (ericelliott) [inline package.json]: Missing another comma? =)

**Comment by therealklanni** [comment]
Crap, what happened there? lol

**Comment by therealklanni** [comment]
Oh, because in mine I've already removed grunt as a dependency. Oops.

**Comment by therealklanni** [comment]
`package.son`? Stupid OS X autocorrect. :frowning:

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  entry: "./source/index.js",
```
Comment (ericelliott) [inline webpack.config.js]: No such file.

We need `./source/browser.js` and `./source/server.js`.

Create a new file called `cuid.js`. Make it export a function that takes an options object. Browser and Server should do this, respectively:

```
import createCuid from './cuid.js';
import fingerprint from './browser.js';

export default () => createCuid({ fingerprint });
```

```js
import createCuid from './cuid.js';
import fingerprint from './server.js';

export default () => createCuid({ fingerprint });
```

Comment (therealklanni) [inline webpack.config.js]: Yep, they exist in mine. I was just adding the webpack config premturely. Feel free to leave this PR open until the file exist (i.e. merged from another PR that closes #25)

Working on #25 currently.

**Comment by ericelliott** [comment]
Awesome! =)

**Comment by therealklanni** [comment]
I'm getting `ERROR in Loader /Users/klanni/Projects/cuid/node_modules/babel/index.js didn't return a function` now, and I'm not sure what it means. If you've run into this before, maybe you can help. I'm leaving it alone for now.

**Comment by ericelliott** [comment]
Any ideas about why the build is passing, then?

**Comment by therealklanni** [comment]
Yeah `rm -rf node_modules` fixed it. Weird.

**Pull request merged**